<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">


background-color: rgba(249, 255, 181, 250);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="f_search">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>147</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>오뮤 다예쁨체</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(20, 204, 20);
	background-color: white;
	border: 4px solid rgb(20, 204, 20);
	border-radius: 20px;</string>
    </property>
    <property name="text">
     <string> 노선 정보</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f_exit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>오뮤 다예쁨체</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(251, 86, 7);
	background-color: white;
	border: 4px solid rgb(251, 86, 7);
	border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>종료</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f_inform">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>273</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>오뮤 다예쁨체</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(58, 134, 255);
background-color: white;
border: 4px solid rgb(58, 134, 255);
border-radius: 20px;
</string>
    </property>
    <property name="text">
     <string>역 정보</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>323</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
